interaction-manager/src/lib_components/gamepad.js

```diff
@@ -67,10 +69,12 @@ class Gamepad {
 
         const isAxisAlias = this.mappings.axisAliases.some(alias => options.actions.includes(alias));
 
-        if (options.actions.length > 1 && isAxisAlias) return console.error(`You can't use an axis action in a combination with a button action`);
+        if (options.actions.length > 1 && isAxisAlias) { return console.error(`You can't use an axis action in a combination with a button action`); }

```

Comment:
Are these curly brackets needed?

Reply:
Yes, but not on the same row. The line is over 120 characters and we need the curly brackets to move it on the next line. Otherwise the linter won't allow it. Fixed it.